types/attestation.yaml

```diff
@@ -15,6 +15,23 @@ IndexedAttestation:
       $ref: './primitive.yaml#/Signature'
       description: "The BLS signature of the `IndexedAttestation`, created by the validator of the attestation."
 
+SingleAttestation:
```

Comment:
we might wanna put this into electra namespace as no namespace types are either phase0 or unspec'd types